src/App.js

```diff
@@ -17,8 +18,35 @@ const tabs = [
   },
 ];
 
-const App = () => (
-  <h1>React tabs</h1>
-);
+class App extends React.Component {
+  state = {
+    number: 0,
```

Comment:
I think it would be better to give this property another name. Something like 'currentTabIndex', because the name 'number' gives us no certain information about what's in it.
But it's only my opinion. Everything else is 👍